content/2_Bronze/Intro_Sets.mdx

```diff
@@ -1226,4 +1226,222 @@ make their implementation easier.
+			$\mathcal{O}(N)$
+		</Quiz.Answer>
+		<Quiz.Answer correct>
+			$\mathcal{O}(\log N)$
+			<Quiz.Explanation>
+				Correct.
+			</Quiz.Explanation>
+		</Quiz.Answer>
+	</Quiz.Question>
+	<Quiz.Question>
+		Which of the following is the correct output of this code?
+		```py
+		s = set()
+		s.add(1)
+		s.add(4)
+		s.add(5)
+		s.add(2)
+		print(3 in s)
+		```
+		<Quiz.Answer>
+			$1$
+			<Quiz.Explanation>
+				Incorrect. The output is a `bool`.
+			</Quiz.Explanation>
+		</Quiz.Answer>
+		<Quiz.Answer correct>
+			False
+			<Quiz.Explanation>
+				Correct. $3$ isn't present in `s`.
+			</Quiz.Explanation>
+		</Quiz.Answer>
+		<Quiz.Answer>
+			True
+			<Quiz.Explanation>
+				Incorrect. $3$ isn't present in `s`.
+			</Quiz.Explanation>
+		</Quiz.Answer>
+		<Quiz.Answer>
+			$0$
+			<Quiz.Explanation>
+Incorrect. The output is a `bool`.
+			</Quiz.Explanation>
+		</Quiz.Answer>
+
+    </Quiz.Question>
+    <Quiz.Question>
+    	Which of the following is the correct output of this code?
+    	```py
+    	s = set([4, 1, 2, 1, 3, 2, 3, 1, 3, 1])
+    	print(len(s))
+    	```
+    	<Quiz.Answer>
+    		$1$
+    		<Quiz.Explanation>
+
+    		</Quiz.Explanation>
+    	</Quiz.Answer>
+    	<Quiz.Answer correct>
+    		$4$
+    		<Quiz.Explanation>
+    			Correct. There are $4$ unique elements in `s`.
+    		</Quiz.Explanation>
+    	</Quiz.Answer>
+    	<Quiz.Answer>
+    		$7$
+    		<Quiz.Explanation>
+
+    		</Quiz.Explanation>
+    	</Quiz.Answer>
+    	<Quiz.Answer>
+    		$10$
+    		<Quiz.Explanation>
+    	Sets only store unique elements.
+    		</Quiz.Explanation>
+    	</Quiz.Answer>
+
+    </Quiz.Question>
+    <Quiz.Question>
+    	What is the time complexity of insertions, deletions, and searches in an unordered map of size $N$?
+    	<Quiz.Answer correct>
+    		$\mathcal{O}(1)$
+    		<Quiz.Explanation>
+    			Correct.
+    		</Quiz.Explanation>
+    	</Quiz.Answer>
+    	<Quiz.Answer>
+    		$\mathcal{O}(N)$
+    	</Quiz.Answer>
+    	<Quiz.Answer>
+    		$\mathcal{O}(N \log N)$
+    	</Quiz.Answer>
+    	<Quiz.Answer>
+    		$\mathcal{O}(\log N)$
+    	</Quiz.Answer>
+    </Quiz.Question>
+    <Quiz.Question>
+    	Which of the following is the correct output of this code?
+    	```py
+    	m = {1: 5, 2: 9, 3: 7, 4: 3}
+    	print(m[2])
+    	```
+    	<Quiz.Answer>
+    		$5$
+    		<Quiz.Explanation>
+
+    		</Quiz.Explanation>
+    	</Quiz.Answer>
+    	<Quiz.Answer correct>
+    		$9$
+    		<Quiz.Explanation>
+    			Correct. The value associated with the key $2$ is $9$.
+    		</Quiz.Explanation>
+    	</Quiz.Answer>
+    	<Quiz.Answer>
+    		$2$
+    		<Quiz.Explanation>
+
+    		</Quiz.Explanation>
+    	</Quiz.Answer>
+    	<Quiz.Answer>
+    		$0$
+    		<Quiz.Explanation>
+
+    		</Quiz.Explanation>
+    	</Quiz.Answer>
+
+    </Quiz.Question>
+    <Quiz.Question>
+    	Which of the following is the correct output of this code?
+    	```py
+    	m = dict()
+    	m[1] = 5
+    	m[2] = 9
+    	m[3] = 7
+    	m[4] = 3
```

Comment:
why not direct initialization

Reply:
keeping consistent between langs + familiarizing with mutating dictionaries